macros/GroupData.json

```diff
@@ -1893,6 +1893,7 @@
                             "AuthenticatorResponse",
                             "AuthenticatorAttestationResponse",
                             "AuthenticatorAssertionResponse" ],
+            "methods":    [],
             "properties": [],
             "events":     []
```

Comment:
These empty arrays are actually unnecessary.

Reply:
Please see my PR description where I talk about these.

Reply:
There is some code out there that fails if these aren't included. I learned that the hard way a couple months ago.